prime-router/config/detekt/detekt.yml

```diff
@@ -0,0 +1,796 @@
+build:
```

Comment:
You are mixing work here.  This issue is for the smoke tests.  You should move this detekt work to another issue, so we can play with it and figure out if we want it.